package.json

```diff
@@ -80,10 +80,14 @@
     }
   },
   "lint-staged": {
-    "*.styled.{js,jsx,ts,tsx}": [
+    "!(*styled).{js,jsx,ts,tsx}": [
       "prettier --config .prettierrc --write",
-      "npm run lint:css",
       "eslint --fix"
+    ],
+    "*.styled.{js,jsx,ts,tsx}": [
+      "prettier --config .prettierrc --write",
+      "eslint --fix",
+      "npm run lint:css"
```

Comment:
We have executed these commands concurrently, so, maybe we can do something like that

```javascript
"lint-staged": {
    "*.{js,jsx,ts,tsx}": [
      "prettier --config .prettierrc --write",
      "eslint --fix"
    ],
    "*.styled.{js,jsx,ts,tsx}": [
      "npm run lint:css",
    ]
}
```

For each file we execute prettier and eslint, and for only styled files we are adding stylelint. Check this solution

Reply:
indeed, it makes sense